src/main/webapp/about.jsp

```diff
@@ -1,3 +1,4 @@
+<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8" %>
```

Comment:
Can we use the more concise version of just `<%@ page pageEncoding="UTF-8"%>` instead?

Reply:
Sounds good; I just checked the HTTP response headers from the live server for `features.jsp` and content type is already set to `text/html; charset=UTF-8` so this shouldn't be necessary.

Reply:
Also checked `/admin/adminSessionsPage`, which originally had the `contentType` attribute. It continues to have the correct `content-type` HTTP header after removing the attribute.

Reply:
I guess for documentation purposes:

From what I understand, the actual page encoding and the response headers encoding is fully determined by `pageEncoding` if `contentType` is not specified. And since the default `contentType` is `text/html; charset=ISO-8859-1`, it turns into `text/html; charset=UTF-8` if we set the `pageEncoding`.

Didn't check what version App Engine supports and implements but I think testing it should be good enough, so I guess a source from Oracle Application Server should be good enough.

https://docs.oracle.com/cd/B14099_19/web.1012/b14014/jspnls.htm#i1005593